ironfish-http-api-v2/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "ironfish-http-api-v2",
+  "version": "0.0.0",
+  "description": "HTTP server to interface with Iron Fish accounts",
+  "private": true,
+  "author": "Iron Fish <contact@ironfish.network> (https://ironfish.network)",
+  "repository": "iron-fish/ironfish",
+  "license": "MPL-2.0",
+  "homepage": "https://github.com/iron-fish/ironfish",
+  "keywords": [
+    "http",
+    "api"
+  ],
+  "scripts": {
+    "build": "tsc -b",
```

Comment:
what's the recommended way to dev? Make a change and re-run build and start?

Reply:
For now, yeah we need to re-build and start. I wanted to lay out the groundwork for the server in this PR, but will add auto + hot reloading in another diff.